_posts/2021-02-07-batman-wechsel.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title:  "B.A.T.M.A.N-Wechsel für Testing-Nutzer"
+author: Louis
+date:   2021-02-07 13:32:14 +0200
+---
+
+__Dieser Artikel richtet sich an Nutzer der Testing-Firmware__ [2019.1.2+bremen4](https://wiki.bremen.freifunk.net/Firmware/Changelog#2019-1-2-bremen4), die helfen wollen, das Bremer Freifunk-Netz schneller zu machen.
+
+## Hintergrund
+Eine Website zu öffnen ist für uns meist nur einen Klick entfernt. Technisch gesehen, sind es dutzende kleine Schritte. Damit die Website durch das Freifunk-Netz zu dir geschickt werden kann, muss der Weg über verschiedene Netzwerkgeräte wie Router und Server zu deinem Endgerät bekannt sein. Es muss eine Route existieren. Um dieses "Routing" kümmert sich im Bremer Freifunk-Netz momentan B.A.T.M.A.N in der Version 13. Die ist nicht mehr ganz taufrisch.
+
+Um unter anderem eine Vielzahl neuer Geräte zu unterstützen, wollen wir auf Gluon 2020 wechseln. Gluon ist die Grundlage unsere Bremer Freifunk-Firmware.
+Da diese aber B.A.T.M.A.N v13 nicht mehr unterstützt, müssen wir vorher einen Wechsel auf B.A.T.M.A.N in der Version 15 vollziehen.
+
+## Der Wechsel
+Wir wollen sicherstellen, dass B.A.T.M.A.N v15 bei uns fehlerfrei funktioniert. Deshalb bitten wir dich, deine Testing-Router auf die neue B.A.T.M.A.N-Version umzustellen.
+
+__Hinweis__
+- Remote first.
+Beginne mit Änderungen am entfernten Gerät.
+- B.A.T.M.A.N v15 und v13 sind nicht kompatibel. Sie meshen nicht. Stelle bitte sicher, dass alle Router, die per Kabel oder WLAN mit deinem Router meshen, die gleiche Domain ausgewählt haben.
+
+### Wechsel per Weboberfläche
+
+1. Halte den Reset-Knopf etwa 10 Sekunden gedrückt bis alle Lämpchen kurz aufblinken.
+2. Öffne die Seite _http://192.168.1.1/_
```

Comment:
Hier sollte noch irgendwo stehen, dass man sich per LAN-Kabel mit dem Knoten verbinden sollte.

Und hier könnte auch noch ein Link z.B. auf https://ffhb.de/anleitungen.html#konfigurationsmodus hin, wo der Wechsel in den Konfig-Modus ausführlich beschrieben ist.